boilerplate/openshift/golang-osd-operator-osde2e/standard.mk

```diff
@@ -61,20 +61,20 @@ container-engine-login:
 	@${CONTAINER_ENGINE} login -u="${REGISTRY_USER}" -p="${REGISTRY_TOKEN}" quay.io
 
 ######################
-# Targets used by e2e test harness
+# Targets used by e2e test suite
 ######################
 
 # create binary
-.PHONY: e2e-harness-build
-e2e-harness-build: GOFLAGS_MOD=-mod=mod
-e2e-harness-build: GOENV=GOOS=${GOOS} GOARCH=${GOARCH} CGO_ENABLED=0 GOFLAGS="${GOFLAGS_MOD}"
-e2e-harness-build:
+.PHONY: e2e-binary-build
+e2e-binary-build: GOFLAGS_MOD=-mod=mod
```

Comment:
This is failing CI on this project now since prow is configured to call `make e2e-harness-build`. Is this expected, and what is the plan to fix this on the suite of repos that might now be broken? cc @ritmun 
https://github.com/openshift/boilerplate/commit/5cb9efec537cd5618cfa2652df71f82f9368ba11

Reply:
Here is all the projects that will break by renaming this target https://github.com/search?q=repo%3Aopenshift%2Frelease%20e2e-harness-build&type=code